maintainers/maintainer-list.nix

```diff
@@ -16488,6 +16488,12 @@
     githubId = 1538622;
     name = "Michael Reilly";
   };
+  onahp = {
+    name = "Setephano Noovao";
+    email = "onahp@pm.me";
+    github = "onahp";
+    githubId = 54986259;
+  };
```

Comment:
Please move this to a separate commit preceding the main one, titled `maintainers: add onahp`

Reply:
Thanks! Have just made the commit